apps/server/src/modules/board/uc/card.uc.spec.ts

```diff
@@ -117,6 +117,13 @@ describe(CardUc.name, () => {
 
 				expect(elementService.create).toHaveBeenCalledWith(card, ContentElementType.RICH_TEXT);
 			});
+			it('should call the service to move the element', async () => {
+				const { user, card, element } = setup();
+
+				await uc.createElement(user.id, card.id, ContentElementType.RICH_TEXT, 3);
+
+				expect(elementService.move).toHaveBeenCalledWith(element, card, 3);
+			});
```

Comment:
maybe too much, but we could check also the type
if position is not a number, you can check that the move service is not called